api/services/MachineService.js

```diff
@@ -241,14 +245,26 @@ function _createMachine() {
     .then((config) => {
       const machine = _driver.createMachine({
         name: config.machinesName
-      }).finally(() => {
-        const len = _awaitingMachines.length;
-        for (let i = 0; i < len; i++) {
-          if (_awaitingMachines[i] === machine) {
-            _awaitingMachines.splice(i, 1);
+      })
+        .then((machineCreated) => {
+          _createBrokerLog(machineCreated, 'Created');
+        })
+        .finally(() => {
+          const len = _awaitingMachines.length;
+          for (let i = 0; i < len; i++) {
+            if (_awaitingMachines[i] === machine) {
+              _awaitingMachines.splice(i, 1);
+            }
           }
-        }
-      });
+        })
+        .catch(() => {
+          _createBrokerLog({
```

Comment:
`_createBrokerLog` should take an empty object as parameter if no machine is linked with the log record.